source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/details/index.tsx

```diff
@@ -1,37 +1,55 @@
 import * as React from 'react';
-
-import { translate, InjectedTranslateProps as i18nProps } from 'react-i18next';
 import { compose } from 'recompose';
-import { ProjectAttributes } from '@data/models/project';
-import { ResourceObject } from 'jsonapi-typescript';
-import { PROJECTS_TYPE } from '@data';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import { ProjectResource, ApplicationTypeResource, attributesFor } from '@data';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
 
 interface Params {
-  project: ResourceObject<PROJECTS_TYPE, ProjectAttributes>;
+  project: ProjectResource;
+  applicationType: ApplicationTypeResource;
 }
 
 type IProps =
   & Params
   & i18nProps;
 
+const mapRecordsToProps = (passedProps) => {
+  const { project } = passedProps;
+  const { type, id } = project;
+
+  return {
+    applicationType: q => q.findRelatedRecord({ type, id }, 'type')
```

Comment:
I think we should just pass the project here instead of specifically the type/id

Reply:
this'll help us be resilient to id-management strategies in the future